src/main/kotlin/com/project/goms/domain/studentCouncil/presentation/data/request/OutingBanRequest.kt

```diff
@@ -0,0 +1,8 @@
+package com.project.goms.domain.studentCouncil.presentation.data.request
+
+import org.jetbrains.annotations.NotNull
+
+data class OutingBanRequest(
+    @field:NotNull
+    val status : Boolean
```

Comment:
goms 전반적인 코드들을 보면 : 를 변수명 앞에 붙혔는데 띄우니까 컨벤션이 떨어지는거 같습니다.
```suggestion
    val status: Boolean
```
이렇게 수정하면 좋을거 같아요!